File: java/SolutionTest.java
import org.junit.Test;
import static org.junit.Assert.*;

public class SolutionTest {
    @Test
    public void testFizzBuzz() {
        java.util.List<String> result = Solution.fizzbuzz();
        
        assertEquals(100, result.size());
        assertEquals("1", result.get(0));
        assertEquals("Fizz", result.get(2));
        assertEquals("Buzz", result.get(4));
        assertEquals("FizzBuzz", result.get(14));
    }
}

File: ruby/solution_test.rb
require 'minitest/autorun'
require './solution'

class FizzBuzzTest < Minitest::Test
  def test_fizzbuzz
    result = fizzbuzz
    assert_equal 100, result.length
    assert_equal "1", result[0]
    assert_equal "Fizz", result[2]
    assert_equal "Buzz", result[4]
    assert_equal "FizzBuzz", result[14]
  end
end
